app/overrides/arclight/helper_overrides.rb

```diff
@@ -1,12 +0,0 @@
-# frozen_string_literal: true
-
-# Overrides to Arclight helper methods
-module HelperOverrides
```

Comment:
I have no clue why this isn't necessary anymore, but I tested the functionality and it still works, and also the test is still passing.